EDEngineer/Resources/Data/blueprints.json

```diff
@@ -15087,4 +15087,1324 @@
     ],
     "Grade": 5
   }
+  {
+    "Type": "Sensors",
+    "Name": "Long Range Scanner",
+    "Engineers": [ "Lori Jameson, Bill Turner, Juri Ishmaak, Lei Chung, Tiana Fortune" ],
+    "Ingredients": [
+      {
+        "Name": "Iron",
+        "Size": 1
+      }
+    ],
+    "Grade": 1
+  }

```

Comment:
there is supposed to be a comma after each blueprint :

        [...]
        "Grade": 1
    },
    {
        "Type": "Sensors",
        [...]